### DISCUSSION TITLE
Thumbnail missing in notification

### DISCUSSION THREAD

**Discussion by iloukine** [ReVanced/revanced-discussions#949] [Question] [answered]
Hello,
Ever since I started using Revanced. I have stopped seeing the thumbnail in the notification. I used to see it on Vanced. I have seen threads about this, but haven't seen the question on github. 
I use KWGT to display the thumbnail of whatever media is playing at the time and whenever that is a YouTube video, it is a genetic (low quality) YouTube logo. 
On this topic, if there a way to always have the notification on? Mine only appears in background mode, not in PIP or if the app is open.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
This is caused by a setting in your OS (possible a setting to use detailed or compact notifications), or it's a change made by YT sometime after the very old Vanced was around.

**Reply by Levikaif** [reply]
![Screenshot_20260917_003234_One UI Home.jpg](https://github.com/user-attachments/assets/66590742-f5c9-4482-a2dd-5f4463831e9d)
 


Same i am also facing this issue what should I need to do

**Reply by iloukine** [reply]
The fix was updating to the new patches, but this was 2 years ago. Not having issue currently.

**Reply by Levikaif** [reply]
So do you have any solution for it I am currently using this version ![Screenshot_20260917_004524_YouTube.jpg](https://github.com/user-attachments/assets/89aa5a66-47ab-49d3-99a6-cffc87f2f8ee)
